### PR TITLE
Added BinarySearch.py

### PR DESCRIPTION
**Pull request by risacodes**
#7 
Added Binary Search in Python.

### CODE REVIEW

File: BinarySearch.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Returns index of x in arr if present, else -1
+def binary_search(arr, low, high, x):
+ 
+    # Check base case
+    if high >= low:
+
```
Comment (anantcodes) [inline BinarySearch.py:6]: Please remove all extraneous spaces.

File: BinarySearch.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ 
+        # If element is present at the middle itself
+        if arr[mid] == x:
+            return mid
+ 
+        # If element is smaller than mid, then it can only
+        # be present in left subarray
+        elif arr[mid] > x:
+            return binary_search(arr, low, mid - 1, x)
+
```
Comment (anantcodes) [inline BinarySearch.py:17]: Here as well. Please remove these spaces for better formatting.

**Review by anantcodes — CHANGES_REQUESTED** [review]
Please remove these spaces. Thanks!

File: BinarySearch.py (hunk 1 above)
Comment (risacodes) [reply BinarySearch.py:6]: Okay, I will do that.

File: BinarySearch.py (hunk 2 above)
Comment (risacodes) [reply BinarySearch.py:17]: Sure

**Comment by risacodes** [comment]
Changes have been made. Please  verify.

**Comment by anantcodes** [comment]
Lgtm

**Pull request merged**